src/main/java/Game.java

```diff
@@ -0,0 +1,64 @@
+import java.awt.*;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Game {
+
+    void play() throws IOException {
+        Hint hint = new Hint();
+        Choice choice = new Choice();
+        Number number = new Number();
+        Restriction restriction = new Restriction();
+        InputStreamReader input = new InputStreamReader(System.in);
+        BufferedReader bufferedReader = new BufferedReader(input);
+
+        System.out.println("게임을 시작합니다.");
+        boolean newGame = true;
+        String answer = null;
+
+        baseballGame :
```

Comment:
```suggestion
        baseballGame:
```
인텔리제이의 코드 자동 정렬을 수행하게 되면, 위와 같이 ``:``를 반복문 이름에 붙여쓰는 걸로 변경됩니다. 이클립스에서 코드 자동 정렬 기능을 사용해보신 것으로 알고있습니다. 인텔리제이 또한 그런 기능이 존재합니다. 
코드 자동 정렬을 수행하는 단축키는 **윈도우 기준**: ``Ctrl + Alt + L `` 입니다. 

이외에도 인텔리제이에는 유용한 단축키들이 많이 있습니다. 이를 많이 알면 알수록 작업 효율도 높아질 것입니다. 처음부터 이런 기능들을 모두 정리하고 단축키를 외우는 것도 하나의 방법이지만, 제가 추천드리는 방법은 인텔리제이의 **Key Promoter X** 플러그인을 설치하여 사용해보시는 걸 추천드립니다. 사용한 기능들의 단축키를 하단 중앙부에 알려주는 기능인데, 단축키 암기에 도움을 많이 받을 수 있으리라 생각됩니다.

Reply:
Key Promoter X 플러그인은 현재 사용중이긴 한데, 단축키를 익히는데에는 아직 시간이 좀 더 필요한 것 같습니다. 감사합니다.